lib/katello/repository_types/README.md

```diff
@@ -28,24 +60,45 @@ Katello::RepositoryTypeManager.register(::Katello::Repository::YUM_TYPE) do
 ```
 This declares the the repository type, specifying a type identifier (e.g. 'yum')
 
-### Service class
+### Service class and friends
 
 ```ruby
-  service_class Katello::Pulp::Repository::Yum
+  pulp3_service_class Katello::Pulp3::Repository::Yum
+  pulp3_api_class Katello::Pulp3::Api::Yum
+  pulp3_plugin 'rpm'
+  ...
 ```
 
-Identifies the service class that is used to interface with pulp 2 and includes methods for creating and updating the type of repository
+Identifies the service class that is used to interface with Pulp and includes methods for creating and updating the type of repository. Also notes the location of the main API class and the name of the corresponding Pulp plugin.
+There are a number of other classes that need to be defined, so follow the example above to ensure that the new type has
+all the neccessary information.
```

Comment:
today I learned: https://stackoverflow.com/questions/41513565/markdown-and-line-wrapping

:+1: